pipeline/metadata/4.3.yaml

```diff
@@ -0,0 +1,534 @@
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-1
+- name: "test-upgrade-ceph-custom-name"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_ceph-custom-name.yaml"
+  global-conf: "conf/nautilus/upgrades/upgrades.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-2
+- name: "test-upgrade-disk-scenarios-container"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_test-disk-scenarios-container.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-2_upgrade.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-2
+- name: "test-upgrade-filestore-to-bluestore"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_test-filestore-to-bluestore.yaml"
+  global-conf: "conf/nautilus/upgrades/upgrades.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-2
+- name: "test-upgrade-skip-tags-wait-for-osds"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_skip-tags-wait-for-osds.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-2_upgrade_skip-tags-wait-for-osds.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-2
+- name: "test-rgw-multisite-primary-to-secondary-tier-1-extd"
+  suite: "suites/nautilus/rgw/tier-1-extn_rgw_multisite-primary-to-secondary.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
+- name: "test-upgrade-rpm-to-container-lvm"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_test-rpm-to-container-lvm.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-2_upgrade.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
```

Comment:
Upgrades is not an FG in itself. So this test suite can contain dmfg also as another tag along with upgrades.

Reply:
Done added as

- upgrades
- dmfg